AndroidConsumer/app/src/androidTest/java/com/karumi/androidconsumer/ExampleUiTest.kt

```diff
@@ -0,0 +1,13 @@
+package com.karumi.androidconsumer
+
+import com.karumi.kotlinsnapshot.KotlinSnapshot
+import org.junit.Assert
+import org.junit.Test
+
+class ExampleUiTest {
+    @Test
+    fun can_resolve_KotlinSnapshot() {
+        val snapshot = KotlinSnapshot()
+        Assert.assertNotNull(snapshot)
```

Comment:
You are not testing `KotlinSnapshot` is working, you only test the dependency is there.

can you change the test for something like this? `14.matchWithSnapshot()` 🎉